myuw/templates/handlebars/card/seattle_application.html

```diff
@@ -0,0 +1,116 @@
+{% load templatetag_handlebars %}
+{% tplhandlebars "seattle_application_card" %}
+<div class="myuw-card application-card">
+{{#if is_returning}}
+    <h3 class="myuw-card-title">
+        Your Returning Student Application
+    </h3>
+
+    <div class="myuw-card-section last">
+		<h4 class="myuw-card-heading myuw-header-msg"> For application status, contact The Office of the University Registrar</h4>
+			<div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Email</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content"><a href="mailto:regoff@uw.edu">regoff@uw.edu</a></div>
+                </div>
+            </div>
+			<div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Phone</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">(206) 543-4000</div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Fax</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">(206) 616-3222</div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Location</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">225 Schmitz Hall</div>
+                </div>
+            </div>
+            <div class="row last">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Mail</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">Registration and Transcripts Office<br>University of Washington <br>Box 355850 <br>Seattle, WA 98195-5850</div>
+                </div>
+            </div>
+	</div>
+	<div class="myuw-card-section" style="padding-top: 8px;">
+		<div>
```

Comment:
Not sure, but it doesn't seem like this <div> is used for anything.

Reply:
it overwrites the padding to look more consistent between chunks.

Reply:
Sorry, github ate my formatting. I was referring to that `<div>`.